src/mocks/recursionTest/import.lua

```diff
@@ -0,0 +1,16 @@
+local Importer = require(script.Parent.Parent.Parent.Importer)
+local bind = require(script.Parent.Parent.Parent.bind)
+
+local importer = Importer.new(script.Parent)
+
+local module = {
+	setConfig = bind(importer, importer.setConfig),
+}
+
+-- Allows this module to be called as import(), otherwise we'd be writing import.import()
+return setmetatable(module, {
+	__call = function(_, path, exports)
+		local caller = getfenv(2).script
+		return importer:import(caller, path, exports)
+	end
+})
```

Comment:
Curious why we need this file. Would it be possible to explore an avenue where we require the import module itself so we're not duplicating code?

Reply:
youre right, that was an artifact of some assumptions Ii made about the test suite needing to avoid require loops. fixed🙏